kernel/src/main/java/org/kframework/kompile/Kompile.java

```diff
@@ -284,6 +284,9 @@ private static Module excludeModulesByTag(Set<String> excludedModuleTags, Module
         DefinitionTransformer numberSentences = DefinitionTransformer.fromSentenceTransformer(NumberSentences::number, "number sentences uniquely");
         Function1<Definition, Definition> resolveConfigVar = d -> DefinitionTransformer.fromSentenceTransformer(new ResolveFunctionWithConfig(d, false)::resolveConfigVar, "Adding configuration variable to lhs").apply(d);
         Function1<Definition, Definition> resolveIO = (d -> Kompile.resolveIOStreams(kem, d));
+        Function1<Definition, Definition> markExtraConcreteRules = d -> DefinitionTransformer.fromSentenceTransformer((m, s) ->
```

Comment:
This was a step from kprove that now needs to happen in kompile. (since we don't allow any modification to definition.kore)
All it does is add the attribute `concrete` to a specific set of rules that get interpreted differently by the backend.
This feature is used extensively by the EVM semantics.
It is now tested by `regression-new/concrete-function`